Gemfile.lock

```diff
@@ -44,61 +44,60 @@ GEM
       jsonapi-renderer (>= 0.1.1.beta1, < 0.3)
     active_storage_base64 (1.0.0)
       rails (~> 6.0)
-    activeadmin (2.3.1)
+    activeadmin (2.6.0)
       arbre (~> 1.2, >= 1.2.1)
       formtastic (~> 3.1)
       formtastic_i18n (~> 0.4)
       inherited_resources (~> 1.7)
       jquery-rails (~> 4.2)
       kaminari (~> 1.0, >= 1.0.1)
-      railties (>= 5.0, < 6.1)
+      railties (>= 5.2, < 6.1)
       ransack (~> 2.1, >= 2.1.1)
       sassc-rails (~> 2.1)
       sprockets (>= 3.0, < 4.1)
-      sprockets-es6 (~> 0.9, >= 0.9.2)
-    activejob (6.0.0)
-      activesupport (= 6.0.0)
+    activejob (6.0.2.1)
+      activesupport (= 6.0.2.1)
       globalid (>= 0.3.6)
-    activemodel (6.0.0)
-      activesupport (= 6.0.0)
-    activerecord (6.0.0)
-      activemodel (= 6.0.0)
-      activesupport (= 6.0.0)
-    activestorage (6.0.0)
-      actionpack (= 6.0.0)
-      activejob (= 6.0.0)
-      activerecord (= 6.0.0)
+    activemodel (6.0.2.1)
+      activesupport (= 6.0.2.1)
+    activerecord (6.0.2.1)
+      activemodel (= 6.0.2.1)
+      activesupport (= 6.0.2.1)
+    activestorage (6.0.2.1)
+      actionpack (= 6.0.2.1)
+      activejob (= 6.0.2.1)
+      activerecord (= 6.0.2.1)
       marcel (~> 0.3.1)
-    activesupport (6.0.0)
+    activesupport (6.0.2.1)
       concurrent-ruby (~> 1.0, >= 1.0.2)
       i18n (>= 0.7, < 2)
       minitest (~> 5.1)
       tzinfo (~> 1.1)
-      zeitwerk (~> 2.1, >= 2.1.8)
+      zeitwerk (~> 2.2)
     addressable (2.7.0)
       public_suffix (>= 2.0.2, < 5.0)
-    airbrussh (1.3.4)
+    airbrussh (1.4.0)
       sshkit (>= 1.6.1, != 1.7.0)
-    annotate (3.0.2)
+    annotate (3.0.3)
       activerecord (>= 3.2, < 7.0)
-      rake (>= 10.4, < 13.0)
+      rake (>= 10.4, < 14.0)
     apipie-rails (0.5.16)
       rails (>= 4.1)
     arbre (1.2.1)
       activesupport (>= 3.0.0)
     ast (2.4.0)
     aws-eventstream (1.0.3)
-    aws-partitions (1.219.0)
-    aws-sdk-core (3.68.0)
+    aws-partitions (1.258.0)
+    aws-sdk-core (3.86.0)
       aws-eventstream (~> 1.0, >= 1.0.2)
-      aws-partitions (~> 1.0)
+      aws-partitions (~> 1, >= 1.239.0)
       aws-sigv4 (~> 1.1)
       jmespath (~> 1.0)
-    aws-sdk-kms (1.24.0)
-      aws-sdk-core (~> 3, >= 3.61.1)
+    aws-sdk-kms (1.27.0)
+      aws-sdk-core (~> 3, >= 3.71.0)
       aws-sigv4 (~> 1.1)
-    aws-sdk-s3 (1.48.0)
-      aws-sdk-core (~> 3, >= 3.61.1)
+    aws-sdk-s3 (1.60.1)
```

Comment:
### [aws-sdk-s3](https://github.com/aws/aws-sdk-ruby)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.48.0 → 1.60.1

[_[change-log](https://github.com/aws/aws-sdk-ruby/tree/master/gems/aws-sdk-s3/CHANGELOG.md), [source-code](https://github.com/aws/aws-sdk-ruby/tree/master/gems/aws-sdk-s3)_]





<details>
<summary>Commits</summary>

A change of **174** commits. See the full changes on [the compare page](https://github.com/aws/aws-sdk-ruby/compare/v1.48.0...v1.60.1).

These are the first 10 commits:
- (3310909) [Refs #575: Fixed issue in assigning metadata to S3 object th…](https://github.com/aws/aws-sdk-ruby/commit/3310909896af5a19b042bed560d948968536d59a)
- (afccdcd) [Merge pull request #578 from matuag/bug-575](https://github.com/aws/aws-sdk-ruby/commit/afccdcd0feb19b81dd26edad3a58ad37d6a6a3e9)
- (c809048) [Updated the bundled list of supported AWS endpoints.](https://github.com/aws/aws-sdk-ruby/commit/c8090482f3ac93cb7952c4d24ed73de549a38982)
- (cb90b23) [Replaced a 1.9 hash syntax usage with 1.8.7 compat.](https://github.com/aws/aws-sdk-ruby/commit/cb90b231b9193b4e9b7a0fb91f999a7a61bf5634)
- (0db20d3) [Resolved an issue with the AWS::S3 client-side-encryption Ci…](https://github.com/aws/aws-sdk-ruby/commit/0db20d3f7389781948c5379551cb7d6b54f72ef5)
- (6dee34b) [Reverted commit e475224b5 due to a regression with S3 client…](https://github.com/aws/aws-sdk-ruby/commit/6dee34ba47e17e5f4123be49764723abf11b44a0)
- (0d59092) [Tag release v1.48.1](https://github.com/aws/aws-sdk-ruby/commit/0d59092611b568a9b88bb106a52dd2eab0fd11d0)
- (6b345f5) [Added member ordering back into Route53 model to resolve an …](https://github.com/aws/aws-sdk-ruby/commit/6b345f52049ac8963a605fd6fe940a529dce85f6)
- (6c55efb) [Updated specs to resolve some issues with updated rspec vers…](https://github.com/aws/aws-sdk-ruby/commit/6c55efba0393611850df07dcdbf97d8e196f3703)
- (d06c28a) [Convert milliseconds to microseconds before passing to Time:…](https://github.com/aws/aws-sdk-ruby/commit/d06c28a23e3abc8aeaedd3fbd136e9022a546da9)


</details>